tasks/client.yml

```diff
@@ -96,4 +96,28 @@
     force: true
   when: client_mgmt_host == inventory_hostname
   become: true
+
+- name: Copy remote beegfs-client.service to /etc/systemd/system/
+  copy:
+    remote_src: true
+    src: /usr/lib/systemd/system/beegfs-client.service
+    dest: /etc/systemd/system/beegfs-client.service
+  notify: Restart BeeGFS client service
+  become: true
```

Comment:
I don't think you need this copy - any config in /etc/ should override the defaults in /usr.

Reply:
The trouble is there is no equivalent beegfs-client.service under /etc/systemd/system at the moment...

Reply:
Could you just copy a template over to /etc/.. for the bits you need to customise? I did something not too dissimilar here: https://github.com/stackhpc/ansible-role-os-monasca-monitor/blob/master/templates/monasca-monitor.target.j2

Reply:
Thanks for spelling that out... did realise you could do partial override of unit file config like that... I've done what you've asked for but I'm getting this:
```
beegfs-client.service lacks both ExecStart= and ExecStop= setting. Refusing.
```
Do you need to supply an entire section?

Reply:
Does this help?  https://askubuntu.com/questions/659267/how-do-i-override-or-configure-systemd-services

Looks like  `systemctl edit foo` could be better than a template.

Reply:
Hmm, not much fun to do with Ansible though... :( will have to resort to using shell: module...

Reply:
Here's the bit I forgot: you need to put it in /etc/systemd/system/foo.service.d/override.conf

Reply:
Done.